### DISCUSSION TITLE
How to add headers?

### DISCUSSION THREAD

**Discussion by niuhongbin1** [FlareSolverr/FlareSolverr#744] [Q&A]
`import requests
import json

url = "http://localhost:8191/v1"

payload = json.dumps({
  "cmd": "request.get",
  "url": "https://www.amiparis.com/api/commerce/v1/sets/man/t-shirts-polos-man?pageindex=2",
  "headers": {
    'ff-country': 'FR',
    'ff-currency': 'EUR'
  },
  "maxTimeout": 60000
})
headers = {
  'Content-Type': 'application/json'
}
    
response = requests.post(url, headers=headers, data=payload)

`print(response.json()['solution']['response'])

Q:  i want to sent url with specify headers like ff-country and ff-currency. But my payload that i write on the top is not working. How should i do to add specify headers with the url?

**Comment by ngosang** [comment]
It's not supported for now.
https://github.com/FlareSolverr/FlareSolverr/issues/266

**Reply by niuhongbin1** [reply]
Tanks for your answer 😸
